### ISSUE TITLE
Microsoft.CodeAnalysis.Workspaces.MSBuild nuget package is missing reference to Microsoft.Extensions.Logging

### ISSUE THREAD

**Issue by akoeplinger** [dotnet/roslyn#72938]
**Version Used**: 4.10.0-3.24202.15

In https://github.com/dotnet/hotreload-utils/pull/379 we hit an exception when upgrading from 4.9 to 4.10 Roslyn:

```
System.IO.FileNotFoundException: Could not load file or assembly 'Microsoft.Extensions.Logging, Version=8.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'. The system cannot find the file specified.

File name: 'Microsoft.Extensions.Logging, Version=8.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'
   at Microsoft.CodeAnalysis.MSBuild.DiagnosticReporterLoggerProvider.CreateLoggerFactoryForDiagnosticReporter(DiagnosticReporter reporter)
   at Microsoft.CodeAnalysis.MSBuild.MSBuildWorkspace..ctor(HostServices hostServices, ImmutableDictionary`2 properties)
   at Microsoft.CodeAnalysis.MSBuild.MSBuildWorkspace.Create(IDictionary`2 properties, HostServices hostServices)
   at Microsoft.CodeAnalysis.MSBuild.MSBuildWorkspace.Create(IDictionary`2 properties)
   at Microsoft.DotNet.HotReload.Utils.Generator.BaselineProject.PrepareMSBuildProject(Config config, EditAndContinueCapabilities capabilities, CancellationToken ct)
   at Microsoft.DotNet.HotReload.Utils.Generator.BaselineProject.Make(Config config, EditAndContinueCapabilities capabilities, CancellationToken ct)
   at Microsoft.DotNet.HotReload.Utils.Generator.Runner.SetupBaseline(EditAndContinueCapabilities capabilities, CancellationToken ct)
   at Microsoft.DotNet.HotReload.Utils.Generator.Runner.Run(CancellationToken ct)
```

Looking at the .nuspec of Microsoft.CodeAnalysis.Workspaces.MSBuild I saw that it is referencing only Microsoft.Extensions.Logging.Abstractions, but not Microsoft.Extensions.Logging.

The latter assembly is referenced since https://github.com/dotnet/roslyn/commit/19751d7131bf2bee3013df2acef9a36b90ebbbdb because `LoggerFilterOptions` is contained in Microsoft.Extensions.Logging, not .Abstractions:
https://github.com/dotnet/roslyn/blob/70316b7a46701e985affc31cd71a0308cd097a28/src/Workspaces/Core/MSBuild/MSBuild/DiagnosticReporterLoggerProvider.cs#L33

We presumably need to update the package dependency here: https://github.com/dotnet/roslyn/blob/8fffb1bcdd5622db9b9734cfcd3f3877c07bcbde/src/Workspaces/Core/MSBuild/Microsoft.CodeAnalysis.Workspaces.MSBuild.csproj#L28-L30

/cc @jasonmalinowski @dibarbet

**Comment by CyrusNajmabadi** [comment]
tagging @JoeRobich as well.

**Comment by akoeplinger** [comment]
Ping. I can send a PR to fix the nuget reference but I'm not sure depending on the package was intentional.

**Comment by dibarbet** [comment]
Believe this was fixed by https://github.com/dotnet/roslyn/pull/73393, should be in the 17.11 packages

**Comment by akoeplinger** [comment]
I can confirm this is fixed in 17.11 packages
